src/main/java/uk/ac/ox/oxfish/experiments/tuna/Runner.java

```diff
@@ -0,0 +1,284 @@
+/*
```

Comment:
This is a replacement of BatchRunner right?
It's cool!

Reply:
Thanks. I wrote with "tuna needs" in mind, but I think it's general enough to be used elsewhere too...